execution/aggregate/scalar_table.go

```diff
@@ -185,8 +185,8 @@ func makeAccumulatorFunc(expr parser.ItemType) (newAccumulatorFunc, error) {
 				AddFunc: func(v float64, _ *histogram.FloatHistogram) {
 					if !hasValue {
 						value = v
-					} else {
-						value = math.Max(value, v)
+					} else if value < v || math.IsNaN(value) {
```

Comment:
Shall we do the same for `min`?

Reply:
+1. I feel we need to apply the same for min